sufia-models/app/models/concerns/sufia/works/metadata.rb

```diff
@@ -0,0 +1,101 @@
+module Sufia::Works
+  module GenericWork
+    module Metadata
+      extend ActiveSupport::Concern
+
+      included do
+
+        property :label, predicate: ActiveFedora::RDF::Fcrepo::Model.downloadFilename, multiple: false
+
+        property :depositor, predicate: ::RDF::URI.new("http://id.loc.gov/vocabulary/relators/dpt"), multiple: false do |index|
+          index.as :symbol, :stored_searchable
+        end
+
+        property :arkivo_checksum, predicate: ::RDF::URI.new('http://scholarsphere.psu.edu/ns#arkivoChecksum'), multiple: false
```

Comment:
`arkivo_checksum` is on the GenericFile in master, and so it may make sense to keep that at the GF (rather than GW) level.